src/main/resources/org/jenkinsci/plugins/github_branch_source/Messages.properties

```
@@ -13,7 +13,7 @@ ForkPullRequestDiscoveryTrait.nobodyDisplayName=Nobody
 GitHubBranchFilter.DisplayName=GitHub Branch Jobs Only
 
 GitHubBuildStatusNotification.CommitStatus.Good=This commit looks good
-GitHubBuildStatusNotification.CommitStatus.Unstable=This commit has test failures
+GitHubBuildStatusNotification.CommitStatus.Unstable=This commit failed tests or other validations
```

pretty sure it is always the same for both singular and plural unless there is some callout to different types. Maybe that is the British thing and the American is different?

```suggestion
GitHubBuildStatusNotification.CommitStatus.Unstable=This commit failed tests or other validation
```